src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -207,13 +213,13 @@ private bool GetCanIncremental(string configHash, string versionName)
             {
                 return false;
             }
-            var version = LastBuildInfo.Versions.SingleOrDefault(v => v.VersionName == versionName);
-            if (version == null)
+            _lbv = LastBuildInfo.Versions.Single(v => v.VersionName == versionName);
+            if (_lbv == null)
```

Comment:
should be SingleOrDefault

---
In reply to: [79977485](https://github.com/dotnet/docfx/pull/724#discussion_r79977485) [](ancestors = 79977485)